androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/utils/ExecuteAfterTimeout.kt

```diff
@@ -0,0 +1,27 @@
+package com.profusion.androidenhancedvideoplayer.utils
+
+import android.util.Log
+import kotlinx.coroutines.CancellationException
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+
+private val TAG = "EXECUTE_AFTER_TIMEOUT"
+
+fun executeAfterTimeout(
+    scope: CoroutineScope,
+    job: Job?,
+    timeInMillis: Long = 650L,
```

Comment:
This shouldn't have a default
```suggestion
    timeInMillis: Long,
```